src/__tests__/Tooltip.spec.js

```diff
@@ -83,7 +83,7 @@ describe('Tooltip', () => {
 
     const tooltips = document.getElementsByClassName('tooltip');
 
-    expect(wrapper.find('.tooltip.show').hostNodes().length).toBe(0);
+    expect(wrapper.find('.tooltip.show').hostNodes().length).toBe(1);
```

Comment:
I *think* this is actually the desired behavior. before the PopperContent was creating a div manually and then mounting into that div. Now it's wrapping the content in the div while it creates the portal which means we do have a host node, but the dom ends up being the same as before.